lib/Optimizer/GraphOptimizer.cpp

```diff
@@ -855,6 +855,41 @@ static void optimizeBatchNorm(Function *F) {
   } // For all nodes in the graph.
 }
 
+/// Check if the input of this method (vector of NodeValues) can be 
+/// legal input of CreateConcat function.
+/// \returns the dimension where the Concat will take place. If not 
+/// concat-able, return -1.
+size_t checkInputsForConcat(std::vector<NodeValue> inputs) {
+  auto firstDims = inputs.front().dims();
+  auto dimSize = firstDims.size();
+  auto index = -1;
```

Comment:
prefer auto only if it improves readability (shorter in most of the cases). In case of ints etc, the explicit full type should be good.

Reply:
done